lib/builders/response_builder.rb

```diff
@@ -52,7 +52,7 @@ def with_status_message(status_message)
       # @param transformer name [String]
       # @return [ResponseBuilder] response builder for chaining
       def with_transformer(transformer)
-        @options[:transformer] = transformer
+        @options.key?(:transformers) ? @options[:transformers] << transformer : @options[:transformers] = [transformer]
```

Comment:
This line is rather dense. Do you mind breaking this up into something like:
```
options[:transformer] ||= []
options[:transformer] << transformer
```

Reply:
Yep, no worries, updated as requested.